### ISSUE TITLE
Parse Issue (Xcode): Module 'better_player' not found

### ISSUE THREAD

**Issue by sanjaysharmajw** [jhomlala/betterplayer#1310]
Launching lib/main.dart on iPhone 11 in debug mode...
Running pod install...
Running Xcode build...
Xcode build done.                                           35.4s
Failed to build iOS app
Parse Issue (Xcode): Module 'better_player' not found
/Users/sanjaysharma/Documents/Projects/GeoC%20Projects/Geo_C_Rider%20/geo_c_rider/ios/Runner/GeneratedPluginRegistrant.m:11:8

Could not build the application for the simulator.
Error launching application on iPhone 11.

**Comment by Sumit258000** [comment]
any updates on this? @sanjaysharmajw @jhomlala ?

**Comment by sanjaysharmajw** [comment]
> any updates on this? @sanjaysharmajw @jhomlala ?

Yes I found the solution. Uninstall the pod file and reinstall it. Before that delete the pod file.

**Comment by OokanNumber1** [comment]
[This](https://github.com/flutter/flutter/issues/43986#issuecomment-803604689) helped me.

**Comment by sanjaysharmajw** [comment]
> [This](https://github.com/flutter/flutter/issues/43986#issuecomment-803604689) helped me.

**Use this in podfile.lock or copy and paste. If you are not able to solve it, you can solve it by coming to TeamView. if you are comfortable**


[Podfile.lock.zip](https://github.com/user-attachments/files/16437346/Podfile.lock.zip)

**Comment by OokanNumber1** [comment]
Thank you so much.

**Comment by jhomlala** [comment]
I think this is no longer happening. If I'm wrong, please create new ticket with some reproducible example.
